app/models/search-result.ts

```diff
@@ -123,6 +134,16 @@ export default class SearchResultModel extends Model {
         return null;
     }
 
+    get isContainedByTitleAndUrl() {
+        if (this.isContainedBy) {
+            return {
+                title: this.resourceMetadata.isContainedBy?.[0]?.title?.[0]?.['@value'],
+                absoluteUrl: this.resourceMetadata.isContainedBy?.[0]?.['@id'],
```

Comment:
You shouldn't need the `?`s on isContainedBy because you're checking for the truthiness in the if statement above.